dask/dataframe/io/tests/test_sql.py

```diff
@@ -332,3 +335,98 @@ def test_no_character_index_without_divisions(db):
     # attempt to read the sql table with a character index and no divisions
     with pytest.raises(TypeError):
         read_sql_table("test", db, npartitions=2, index_col="name", divisions=None)
+
+
+def test_to_sql():
+    df_by_age = df.set_index("age")
+    df_appended = pd.concat([df, df,])
+
+    # Set this to False to use a sqlite DB file under the current directory, and not delete it (for easier debugging
+    # after tests have run)
+    rm_tmpdir = True
+    if rm_tmpdir:
+
+        @contextmanager
+        def tmp():
+            with tmpfile() as f:
+                yield "sqlite:///%s" % f
+
+    else:
+        from pathlib import Path
+
+        path = Path.cwd() / "tmp" / "sqlite"
```

Comment:
Certainly shouldn't be in the CWD, and some of us here **hate** pathlib!

Reply:
gtk! just a simple call to `tmpfile()` now!